<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mirall::ProtocolWidget</class>
 <widget class="QWidget" name="Mirall::ProtocolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sync Activity</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="_treeWidget">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>4</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDialogButtonBox" name="_dialogButtonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::NoButton</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
